monai/transforms/spatial/dictionary.py

```diff
@@ -607,6 +765,7 @@ def __init__(
         padding_mode: GridSamplePadModeSequence = GridSamplePadMode.REFLECTION,
         as_tensor_output: bool = True,
         device: Optional[torch.device] = None,
+        dtype: Union[DtypeLike, torch.dtype] = np.float32,
```

Comment:
Seems the default value doesn't match doc-string.

Thanks.

Reply:
thanks that's a good point, I reviewed the dtype options, they should be good now. float32 is good as a data augmentation default...